community-plugins.json

```diff
@@ -6697,5 +6697,12 @@
     "author": "celeste",
     "description": "Loom in Obsidian",
     "repo": "cosmicoptima/loom"
+  },
+  {
+    "id": "personal-assistant",
+    "name": "Personal Assistant",
+    "author": "edony",
+    "description": "Obsidian management tool for plugins, notes and shortcuts which help you to use Obsidian efficiently.",
```

Comment:
We have recently published a guide for plugin descriptions to follow here: <https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple> I suggest you reword the description to more accurately capture what this plugin does. You should also not need to mention "Obsidian" in the description since that information is already a given.

Let me know once you've updated the description to something matching our guidelines. Thanks!

Reply:
@liamcain Thank you for your comments. It's great suggestion about summarizing the scope of plugin, I think it's some streamlined workflows of plugins and memos management to improve efficiency. And I deleted the command which is not related and update the description of plugin, maybe I will startup a new plugin. ^-^

I have already finished a new PR, please review it again.

Reply:
Looks great, thanks!